Link error with embassy-time `std` after ETQU 0.3.1 update

In my app using tokio as executor on a windows host I observe following problem:

Cargo.toml
``` toml
[dependencies]
# ...
embassy-time = { version = "0.5.1", features = ["std", "generic-queue-64"] }
# ...
```

```
error: linking with `link.exe` failed: exit code: 1120
...
  = note: libembassy_executor_timer_queue-bf561fbb86a3a64d.rlib(embassy_executor_timer_queue-bf561fbb86a3a64d.embassy_executor_timer_queue.1e351648231269a7-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __embassy_time_queue_item_from_waker referenced in function _ZN28embassy_executor_timer_queue14TimerQueueItem18from_embassy_waker17hf61ae3dbb1e813d9E
          C:\myapp.exe : fatal error LNK1120: 1 unresolved externals
```

When I pin ETQU with `cargo update -p embassy-time-queue-utils --precise 0.3.0` everything builds fine again.

To me this looks like for whatever reason the new ETQU tries to use the integrated timer queue instead of a generic one as selected by the `embassy-time` feature.

--- a/embassy-time-queue-utils/CHANGELOG.md
+++ b/embassy-time-queue-utils/CHANGELOG.md
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- next-header -->
 ## Unreleased - ReleaseDate
 
+- By default, enabling a `generic-queue-*` feature disables the integrated timers. Integrated timers
+  can be enabled alongside the generic queue by using the `integrated-timers` feature.
+
 ## 0.3.1 - 2026-04-16
 
 - Fixed an issue where never-ending timers were not correctly removed from the timer queue

--- a/embassy-time-queue-utils/Cargo.toml
+++ b/embassy-time-queue-utils/Cargo.toml
@@ -28,16 +28,23 @@ embassy-executor-timer-queue = { version = "0.1", path = "../embassy-executor-ti
 #! ### Generic Queue
 
 #! By default this crate uses a timer queue implementation that is faster but depends on `embassy-executor`.
-#! It will panic if you try to await any timer when using another executor.
-#! 
+#! It will fail to build if you try to use it without an embassy executor in the project.
+#!
 #! Alternatively, you can choose to use a "generic" timer queue implementation that works on any executor.
-#! To enable it, enable any of the features below.
-#! 
+#! To enable it, enable any of the `generic-queue-*` features below. Note that these features by themselves
+#! replace the integrated queue implementation.
+#!
 #! The features also set how many timers are used for the generic queue. At most one
 #! `generic-queue-*` feature can be enabled. If none is enabled, a default of 64 timers is used.
 #!
-#! When using embassy-time-queue-driver from libraries, you should *not* enable any `generic-queue-*` feature, to allow the
-#! end user to pick.
+#! If both options are enabled, the integrated queue implementation will be used by default as the
+#! `Queue` implementation. Both implementations will be separately available under their own modules.
+#!
+#! When using embassy-time-queue-utils from libraries, you should *not* enable any `generic-queue-*`
+#! feature, to allow the end user to pick.
+
+## Enable the integrated queue implementation _even when generic queues are also enabled_.
+integrated-timers = []
 
 ## Generic Queue with 8 timers
 generic-queue-8 = ["_generic-queue"]

--- a/embassy-time-queue-utils/src/lib.rs
+++ b/embassy-time-queue-utils/src/lib.rs
@@ -5,11 +5,16 @@
 use core::task::Waker;
 
 pub mod queue_generic;
+
+// Not explicitly enabled - acts as default, but cannot be relied upon as an implementation detail.
+#[cfg(all(not(feature = "integrated-timers"), not(feature = "_generic-queue")))]
+mod queue_integrated;
+#[cfg(feature = "integrated-timers")]
 pub mod queue_integrated;
 
-#[cfg(feature = "_generic-queue")]
+#[cfg(all(not(feature = "integrated-timers"), feature = "_generic-queue"))]
 type QueueImpl = queue_generic::Queue;
-#[cfg(not(feature = "_generic-queue"))]
+#[cfg(any(feature = "integrated-timers", not(feature = "_generic-queue")))]
 type QueueImpl = queue_integrated::Queue;
 
 /// The default timer queue, configured by the crate's features.

